shoov/modules/custom/shoov_restful/plugins/restful/node/ui_builds/1.0/ShoovUiBuildsResource.class.php

```diff
@@ -31,7 +31,7 @@ public function publicFieldsInfo() {
       'resource' => array(
         'repository' => array(
           'name' => 'repositories',
-          'full_view' => FALSE,
+          'full_view' => TRUE,
```

Comment:
can you please confirm that running UI regression tests still works properly after this change.

Follow [tutorials](http://shoov.io/tutorials/lesson1-install/) if needed